store/boltdb/boltdb.go

```diff
@@ -23,15 +24,28 @@ var (
 	ErrBoltBucketOptionMissing = errors.New("boltBucket config option missing")
 )
 
+const (
+	dbRead  os.FileMode = 0444
+	dbWrite os.FileMode = 0644
+)
+
 //BoltDB type implements the Store interface
 type BoltDB struct {
 	client     *bolt.DB
 	boltBucket []byte
 	dbIndex    uint64
+	path       string
+	timeout    time.Duration
+	// TransientConnection should be  set if the application wants to
+	// open more  than connection to the  Bolt DB. This forces an open
+	// and close for every opertion rather than opening the connection
+	// during New
+	TransientConnection bool
```

Comment:
BoltDB being a file based local DB the natural/intuitive way to operate it open once and use the handle for read/writes. So my preference is to keep that as the default.